src/elements/kw-action-card/kw-action-card.html

```diff
@@ -0,0 +1,72 @@
+<link rel="import" href="../../bower_components/polymer/polymer.html">
+<link rel="import" href="../../bower_components/iron-flex-layout/iron-flex-layout.html">
+
+<link rel="import" href="../kano-style/typography.html">
+<link rel="import" href="../kano-style/button.html">
+
+<dom-module id="kw-action-card">
+    <template>
+        <style>
+            :host a {
+                height: 266px;
+                width: 300px;
+                box-shadow: 0 2px 0 0 rgba(0,0,0,0.06);
+                border-radius: 3px;
+                background-color: white;
+                @apply --layout-vertical;
+                @apply --layout-center-justified;
+                overflow: hidden;
+                transform: translateZ(0);
+                text-decoration: none;
+            }
+            .header {
+                color: #5a5a5a;
+                margin: 0;
+                padding: 0 24px;
+                text-decoration: none;
+                font-size: 18px;
+                text-align: center;
+            }
+            button.action {
+                background-color: rgba(255,255,255,0.7);
+                padding: 11px 23px 10px;
+                font-family: var(--font-body);
+                font-size: 18px;
+                font-weight: 600;
+                color: #5a5a5a;
+                margin: 13px 25px 5px;
+                border: 1px solid #5a5a5a;
+                border-radius: 50px;
+                transition: all linear 200ms;
+            }
+            :host a:hover button.action {
+                background-color: #62b22a;
+                border-color: #62b22a;
+                color: white;
+            }
+        </style>
+        <a href$="[[link]]">
+            <h4 class="header" id="title">[[header]]</h4>
+            <button type="button" class="action">[[label]]</button>
```

Comment:
Is that case, the anchor is the focusable element, the button here doesn't have any action linked to it